tools/topology/sof/pipe-smart-amplifier-playback.m4

```diff
@@ -0,0 +1,82 @@
+# Smart amplifier playback Pipeline
+#
+#  Playback with smart_amp(Smart Amplifier), it will take the feedback(B2) from capture pipeline.
+#
+# Pipeline Endpoints for connection are :-
+#
+#	Playback smart_amp
+#	B1 (DAI buffer)
+#
+#
+#  host PCM_P -- B0 --> smart_amp -- B1--> sink DAI0
+#			   ^
+#			   |
+#			   B2
+#			   |
+
+# Include topology builder
+include(`utils.m4')
+include(`buffer.m4')
+include(`pcm.m4')
+include(`pga.m4')
+include(`smart_amp.m4')
+include(`mixercontrol.m4')
+include(`bytecontrol.m4')
+
+ifdef(`SMART_TX_CHANNELS',`',`errprint(note: Need to define DAI TX channel number for sof-smart-amplifier
+)')
+ifdef(`SMART_FB_CHANNELS',`',`errprint(note: Need to define feedback channel number for sof-smart-amplifier
+)')
+
```

Comment:
can you place an inline comment above these checks (and the other ones in this PR)

Reply:
I removed those "# define"s, as I thought the print error message has already given detail information?